### ISSUE TITLE
IP inversions using Pygimli

### ISSUE THREAD

**Issue by 9Mo9** [gimli-org/gimli#544]
### Problem description

Please describe your issue here.

Hello, I hope that you are doing great

Based on Gunther 2d inversion scripts, I have written this script to invert my data, and then save them into a .vtk file. 
When I open the vtk results I don't find the chargeability; I wanted to know how can I use my script (using pygimli) to make IP inversion.

Python version: 3.8

### Steps to reproduce
--------------------------------------------------------------------------------
```
from pygimli.physics import ert
import pygimli.meshtools as mt
data=ert.load('P1_filtered_new_topo.Dat')
res = ert.ERTManager('P1_filtered_new_topo.Dat')
res.data["k"] = ert.geometricFactors(res.data)
res.estimateError()
mesh = mt.createParaMesh(data.sensorPositions(), paraDX=0.5, paraDepth=20,
                         paraBoundary=10, boundary=4.4, quality=34.5)
# res.fw.blockyModel = True
#inversion with the mesh
#res.invert(mesh=mesh, stopAtChi1=False, lam =20, zWeight = 1, robustData= True, blockyModel=True, verbose=True)

res.invert(stopAtChi1=False, lam =15, zWeight = 1, robustData= True, blockyModel=True, verbose=True)
res.showResult()
# res.invert(blockyModel=True)
res.saveResult()

# res.showResult()
# res.showMisfit()
# res.showMisfit(errorWeighted=True)
```

----------------------------------------------------------------------------------------------------------------------------

Thank you

**Comment by halbmy** [comment]
I just derived an `ERTIPManager` class from the `ERTManager`. It first does the classic DC inversion and then an TD-based IP inversion of the `data["ip"]` data.

To try it, you will have to checkout the current dev version: https://www.pygimli.org/installation.html#staying-up-to-date
before it is integrated in the next version.

**Comment by halbmy** [comment]
Note that there is an example notebook on https://github.com/gimli-org/notebooks/blob/main/IP/TDIP/invertIPmanager.ipynb which will make a pygimli example in the upcoming version v1.4.3

**Comment by 9Mo9** [comment]
Thank you for your answer

**Comment by halbmy** [comment]
The example is online on the dev-channel website and soon available through v1.4.3
http://dev.pygimli.org/_examples_auto/3_induced_polarization/plot_08_TDIPinversion.html

**Comment by 9Mo9** [comment]
Thank you Thomas

Sincerely

Le ven. 30 juin 2023 à 08:11, Thomas Günther ***@***.***> a
écrit :
